packages/forms-web-app/tests/unit/controllers/appellant-submission/application-number.test.js

```diff
@@ -86,7 +86,7 @@ describe('controllers/appellant-submission/application-number', () => {
       });
     });
 
-    it('should redirect to `/appellant-submission/upload-application` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/upload-application` if valid', async () => {
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks